templates/default/sysvinit.service.erb

```diff
@@ -106,6 +109,25 @@ _status_q() {
     _status >/dev/null 2>&1
 }
 
+_wait_for_listening() {
+  echo -n "Waiting for consul daemon to be listening..."
+  for i in `seq 1 30`; do
+    # if ! start-stop-daemon --quiet --stop --test --pidfile $pidfile --user $user; then
+    if ! _status_q; then
```

Comment:
Hey

I was not able to reproduce problem you described with debian 7 (also using kitchen), but I moved sleep any way to beginning of loop.

Also, I fixed indentation.